tests/fast-integration/README.md

```diff
@@ -46,6 +46,24 @@ npm install
 npm test
 ```
 
+
+#### To run eventing specific tests locally, follow these steps:
+1. Install dependencies:
+```bash
+cd kyma/tests/fast-integration
+npm install
+```
+
+2. Define SAP Event Mesh secret file path in `EVENTMESH_SECRET_FILE` environment variable.
```

Comment:
One last suggestion so that there are enough info here, you could mention that the file you are expecting is a json file, and where one could get that file.

Reply:
I have added more info.